addon/components/ownership-selection-modal/people/component.js

```diff
@@ -63,6 +63,27 @@ export default Component.extend({
     }))
   }),
 
+  _reloadSelected: observer('entity', function() {
+    this.get('currentUser').fetchOwnerships().then((ownerships) => {
+      let currentOwnership = ownerships.findBy('id', this.entity.ownedBy);
+      let selectedUsers;
+
+      if (currentOwnership.userIds) {
+        selectedUsers = this.availableUsers.filter((x) => {
+          return currentOwnership.userIds.includes(x.id);
+        });
+      } else {
+        selectedUsers = [];
+      }
+      
+      this.set('selectedUsers', selectedUsers);
+
+      if (isEmpty(this.selectedUsers)) {
+        this.set('currentWindow', 'groups')
```

Comment:
are you sure about this ? isn't currentWindow on the parent component ?

Reply:
yep currentWindow is in the parent component, but not sure how else to switch back to the groups tab if there's no selected users. or should we just be leaving the tab at 'people' even if there's no selected users?

Reply:
if we are going to leave it as it is for the tabs then i'll just remove this